README.md

```diff
@@ -2,17 +2,17 @@
 
 ## Tesseract User Manual
 
-- [User Manual](tessdoc/)
+- [User Manual](http://tesseract-ocr.github.io/tessdoc/)
```

Comment:
Please use https: instead of http: where possible.

Reply:
done via https://github.com/tesseract-ocr/tesseract-ocr.github.io/pull/4